recipes/amplisim/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: amplisim
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/rki-mf1/amplisim
+  git_tag: v{{ version }}
+  sha256: 6697558611dfb8b83f77ebc825f83112e9696eea9218dd35e9dac7705bf9f474
+
+build:
+  number: 0
+  skip: True  # [osx]
+  run_exports:
+    - {{ pin_subpackage('amplisim', max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - make
+  host:
+    - zlib
+    - xz
+    - bzip2
+    - libcurl
+    - libdeflate
+  run:
+    - zlib
+    - xz
+    - bzip2
+    - libcurl
+    - libdeflate
+
+test:
+  commands:
+    - amplisim --help
```

Comment:
Dear @kdm9 ,
thanks again for your feedback. I intended to resolve this by adding a little test suite when I noticed that the bioconda test framework seems to be not okay with my way of using the `test:` section. First and foremost, looking at the error on Azure I hypothesize that the 0-index-argument has to be the tool name which is not always the case in may bash script for testing. Also, on the **Guidelines for bioconda recipes** webpage I found this excerpt

> An adequate test must be included in the recipe. An “adequate” test depends on the recipe, but must be able to detect a successful installation. While many packages may ship their own test suite (unit tests or otherwise), including these in the recipe is not recommended since it may timeout the build system on CircleCI. We especially want to avoid including any kind of test data in the repository.

which indicates that downloading public test data, like it is done by my test script, is also depreciated. Then I scanned through some tools' recipe and figured that most of them are just calling the help page, perhaps grep'ing some substring in there. Hence, I am stuck now what to do to solve your request. Can I 

1. use some other keyword within `test:` to launch a tiny bash test script that itself downloads some (few MB) data? The data is so little a basic GH Action runner can handle it.
2. roll back and just call my help page too? The tool itself has a rather sophisticated and tool-specific CI.  😃

I'd like to circumvent adding more test data to the repository tbh.

Reply:
It's optional to test with some real data, but it does catch non-working packages. As it seems a bit too hard in this case, i'm happy to  accept this package sans tests (just with amplisim --help to check it was installed, like you originally had). However, please at least test the compiled package with a full dataset. You can do 

@BiocondaBot please fetch artifacts 

to ask the bot to send a package link, then follow those instructions.

Reply:
Again, thanks a lot for your assistance. I learned a lot about the contribution mechanisms of conda / bioconda.